### ISSUE TITLE
thread-safety is missing

### ISSUE THREAD

**Issue by yegor256** [volodya-lombrozo/jtcop#436]
I'm running

```
mvn install -T 1C
```

and I'm getting this warning:

```
[WARNING] *****************************************************************
[WARNING] * Your build is requesting parallel execution, but this         *
[WARNING] * project contains the following plugin(s) that have goals not  *
[WARNING] * marked as thread-safe to support parallel execution.          *
[WARNING] * While this /may/ work fine, please look for plugin updates    *
[WARNING] * and/or request plugins be made thread-safe.                   *
[WARNING] * If reporting an issue, report it against the plugin in        *
[WARNING] * question, not against Apache Maven.                           *
[WARNING] *****************************************************************
[WARNING] The following plugins are not marked as thread-safe in eo:
[WARNING]   com.github.volodya-lombrozo:jtcop-maven-plugin:1.3.1
```

Would be great to make this plugin thread-safe.

**Comment by yegor256** [comment]
@volodya-lombrozo please, pay attention

**Comment by volodya-lombrozo** [comment]
@yegor256 Thank you for the info. We will try to fix this soon.

**Comment by volodya-lombrozo** [comment]
@yegor256 I have tried  to reproduce this problem and failed: https://github.com/volodya-lombrozo/jtcop/pull/469 (can't reproduce it)

I just don't see this warning. Can you please share a bit more details about your environment? Are you sure you invoke `jtcop-maven-plugin` with this command?

```
mvn install -T 1C
```
Maybe there is something else that I'm missing?

**Comment by yegor256** [comment]
@volodya-lombrozo I can't reproduce it either, closing
